lib/tasks_private/spec_helper.rake

```diff
@@ -84,24 +101,27 @@ class PopulateTower
   end
 
   def del_obj(uri, match_name)
-    data = JSON.parse(@conn.get(uri).body)
-    data['results'].each do |item|
+    obj = get_obj(uri)
+    obj['results'].each do |item|
       next if item['name'] != match_name
       puts "Deleting old #{item['name']}: #{item['url']}"
       @conn.delete(item['url'])
```

Comment:
Found a bug not related to your changes.  We should exit this method here by `return @conn.delete(item['url'])` since we found our target. Otherwise, Line 112 will still be executed and is simply doing useless work.

Reply:
I've refactored the method so it may be a bit cleaner and better to understand what it does. Tell me if you like it :wink: If not, I can revert it.